Source/WebCore/accessibility/ios/WebAccessibilityObjectWrapperIOS.h

```diff
@@ -56,6 +56,7 @@ static NSString * const UIAccessibilityTextualContextSourceCode = @"UIAccessibil
 - (BOOL)accessibilityCanFuzzyHitTest;
 
 - (BOOL)isAccessibilityElement;
+- (void)_clearCachedIsAccessibilityElementState;
```

Comment:
Maybe internal methods like this should be in a different section and not grouped with the public accessibility methods like isAccessibilityElement and accessibilityLabel?